### DISCUSSION TITLE
Functions which were used in TCPDF, now with issues

### DISCUSSION THREAD

**Discussion by sscherger** [mpdf/mpdf#2021] [General]
Hello,

I am currently in the process of converting our invoices from TCPDF to MPDF. I have not used HTML so far and would like to keep it that way. However, some functions have obviously not been adopted, or perhaps a little differently?

These functions are obviously not defined? Or are there comparable functions in MPDF?
deletePage()
GetPageHeight()
getStringHeight()

These functions do not cause a warning, but work differently:
$mpdf->SetFont(‘Helvetica’,‘U’,8):
The font change is ignored and not adopted, and the text is not underlined, whereas it would be displayed in italics and bold.

Rect():
Background-Color in TCPDF was red with 
Rect(110, 47, 80, 5, ‘F’, 0, array(176, 35, 42));
now it's black, but without a warning

I was already able to solve this one:
GetY() (this one I found out to must be changed to $mpdf->y)

I have also looked for these functions in the manual here:
[https://mpdf.github.io/](https://mpdf.github.io/)
but unfortunately found nothing. Is there perhaps a hidden search function in the manual? 

Kind Regards
Stefan

**Comment by finwe** [comment]
The manual has no search apart from Google.

Methods not mentioned in the manual are either not available or not officially supported (look for them in the souce code but do not rely on them being available in future versions when method scope is defined).
